templates/block/block--image-gallery.html.twig

```diff
@@ -29,17 +29,19 @@
 		{% endif %}
 		{{ title_suffix }}
 		<div class="row row-cols-lg-6 row-cols-md-3 row-cols-2 gallery-div">
-			{% for key, item in content['#block_content'].field_image_gallery_photo if key|first != '#' %}
+			{% for key, item in content['#block_content'].field_image_gallery_photo  %}
+			{% if key|first != '#' %}
 				{% if item.entity.field_media_image_caption.value|render|striptags|trim %}
 					{% set photoDescription = item.entity.field_media_image_caption.value|render|striptags|trim %}
 				{% else %}
 					{% set photoDescription = "" %}
 				{% endif %}
 				<div class="col gallery-item">
-					<a href="{{ file_url(item.entity.field_media_image.entity.fileuri) }}" class="glightbox ucb-gallery-lightbox" data-gallery="gallery{{ content['#block_content'].id() }}" data-glightbox="description: {{ photoDescription }} ">
+					<a href="{{ file_url(item.entity.field_media_image.entity.fileuri) }}" class="glightbox ucb-gallery-lightbox" data-gallery="gallery{{ content['#block_content'].id() }}" data-glightbox="description: yup ">
```

Comment:
What is this?   Surely the description shouldn't be a hard-coded string to "yup"